metadata-ingestion/src/datahub/ingestion/source/dremio/dremio_sql_queries.py

```diff
@@ -242,7 +242,7 @@ class DremioSQLQueries:
         SYS.JOBS_RECENT
     WHERE
         STATUS = 'COMPLETED'
-        AND LENGTH(queried_datasets)>0
+        AND ARRAY_SIZE(queried_datasets)>0
```

Comment:
On Software, the datatype is a varchar, LENGTH should be used.

Reply:
https://github.com/datahub-project/datahub/pull/11817

Reply:
Dremio Software docs suggest queried_datasets is an array of varchar  `[varchar]`-> https://docs.dremio.com/current/reference/sql/system-tables/jobs_recent/#fields

Is the type in the doc incorrect ?

Reply:
For Dremio SW this is a varchar - the docs are incorrect though as it states an array of strings, i.e. [varchar] vs varchar. In SW this is coming through as a varchar (although it is a string that is an array representation), whereas on Cloud this correctly comes through as an array of strings. there is a PR open to fix this [here](https://github.com/datahub-project/datahub/pull/11817)

Reply:
Thanks for confirming. Can you please add this inconsistency of queried_datasets  type as NOTE comment on fix PR ?